cypress/integration/pages/mediaAssetPage/tests.js

```diff
@@ -72,6 +72,27 @@ export const testsThatFollowSmokeTestConfig = ({
         },
       );
     });
+    it('should have href that matches assetURI for 1st related content link', () => {
+      cy.request(`${config[service].pageTypes[pageType].path}.json`).then(
+        ({ body }) => {
+          const arrayLength = body.relatedContent.groups.length;
```

Comment:
I'd avoid ever using generic variable name like this. Maybe something like `numRelatedContentGroups`

Reply:
Ah yes, will change